lib/task_helpers/imports/custom_buttons.rb

```diff
@@ -52,10 +64,13 @@ def add_children(obj, new_obj)
         end
 
         def add_associations(obj, new_obj)
-          if obj['associations'].present?
-            obj['associations'].each do |assoc|
-              new_obj.send("#{assoc.first}=", create_object(*assoc).first)
```

Comment:
To support connecting to an existing dialog by name, change this to:
```
dialog_label = assoc.last.first['attributes'].delete('dialog_label')
ra = create_object(*assoc).first
ra.dialog = Dialog.in_region(MiqRegion.my_region_number).find_by(:label => dialog_label)
new_obj.send("#{assoc.first}=", ra)
```

Reply:
done